src/app/components/settings-area/settings-area.component.ts

```diff
@@ -52,7 +44,46 @@ waveForms=[{name:'sine',symbol:String.fromCharCode(0x223F)},{name:'square',symbo
           this.preset.adsr.attack=this.propSliderAttack.value
         }
       }
+
+      this.propSliderDecay={
+        name:'Decay',
+        min:0.1,
+        max:2,
+        step:0.1,
+        value:this.preset.adsr.decay,
+        action:()=> {
+          this.audioService.setDecay(this.propSliderDecay.value)
+          this.preset.adsr.decay=this.propSliderDecay.value
+        }
+      }
+
+      this.propSliderSustain={
+        name:'Sustain',
+        min:0,
+        max:1,
+        step:0.1,
+        value:this.preset.adsr.sustain,
+        action:()=> {
+          this.audioService.setDecay(this.propSliderSustain.value)
```

Comment:
Error on method call 

```suggestion
          this.audioService.setSustain(this.propSliderSustain.value)
```